easybuild/easyconfigs/t/ToFu/ToFu-1.4.0-intel.patch

```diff
@@ -0,0 +1,13 @@
+--- tofu-1.4.0/setup.py.orig	2019-05-29 00:08:47.000000000 +0200
++++ tofu-1.4.0/setup.py	2019-05-29 00:08:42.000000000 +0200
+@@ -64,8 +64,8 @@
+     os.environ['CC'] = 'gcc-8'
+     os.environ['CXX'] = 'gcc-8'
+ else:
+-    os.environ['CC'] = 'gcc'
+-    os.environ['CXX'] = 'gcc'
++    os.environ['CC'] = 'icc'
++    os.environ['CXX'] = 'icpc'
```

Comment:
A better patch for EasyBuild is (probably) to let it pick up the already defined env vars for CC/CXX, ie to drop everything that tries to define os.environ of CC/CXX in setup.py